sys/include/net/gnrc/netif.h

```diff
@@ -60,6 +63,12 @@
 extern "C" {
 #endif
 
+/**
+ * @brief   A Link Check request was scheduled
+ */
+#define GNRC_NETIF_LORAWAN_FLAGS_LINK_CHECK                (0x00020000U)
```

Comment:
Didn't you want to move to `sys/include/net/gnrc/netif/lorawan.h` as we agreed this is LoRaWAN specific (see https://github.com/RIOT-OS/RIOT/pull/11022#discussion_r341591268)?

Reply:
Also `0x00020000U` doesn't fit into the flag field provided by `gnrc_netif_lorawan_t`. It is only `uint8_t`.

Reply:
Otherwise if you *do* want to `gnrc_netif_t::flags` rather than `gnrc_netif_t::lorawan::flags`, it should go back to `sys/include/net/gnrc/netif/flags.h`, but as it is LoRaWAN specific it really shouldn't.

Reply:
oh wait, I thought that was removed... let me check